pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md

```diff
@@ -46,17 +51,19 @@ Convert it to format that is used internally and split to the train and test par
 
 * Training annotation
 ```bash
-python3 ./model_templates/horizontal-text-detection/tools/create_dataset.py \
-    --config ./model_templates/horizontal-text-detection/datasets/dataset_train.json \
-    --output ${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json
+python ./model_templates/horizontal-text-detection/tools/create_dataset.py \
+    --config ./model_templates/horizontal-text-detection/tools/datasets/dataset_train.json \
+    --output ${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json \
+    --root ./
```

Comment:
I guess that root should be `${DATA_DIR}/text-dataset/`.

Reply:
Fixed